edb/graphql/types.py

```diff
@@ -980,16 +1156,20 @@ def get_order_fields(self, typename):
                     fields[name] = GraphQLInputObjectField(
                         self._gql_ordertypes['Ordering']
                     )
-            elif t.is_object_type():
+            elif isinstance(t, s_objtypes.ObjectType):
                 # It's a link so we need the link's type order input
                 t_name = t.get_name(self.edb_schema)
-                fields[name] = self._gql_ordertypes[t_name]
-
-            # We ignore pointers that aren't scalars or objects.
+                # XXX: why is below not a GraphQLField?
```

Comment:
You are right, this should be wrapped into a GraphQLInputObjectField, just like above.